catalog/nato/stanag4559/nato-stanag4559-common/src/main/java/org/codice/alliance/nato/stanag4559/common/Stanag4559.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.alliance.nato.stanag4559.common;
+
+import java.io.InputStream;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+
+@Path("/")
+public interface Stanag4559 {
+
+    @GET
+    @Produces("text/plain")
```

Comment:
You could use [`MediaType.TEXT_PLAIN`](http://docs.oracle.com/javaee/6/api/javax/ws/rs/core/MediaType.html#TEXT_PLAIN) for these.